src/Padawan/Framework/Domain/Project/Persister.php

```diff
@@ -26,9 +26,10 @@ public function __construct(PathResolver $path, LoopInterface $loop)
     public function save(Project $project)
     {
         $this->checkForPadawanDir($project->getRootFolder());
-        return $this->fs->file(
-            $this->getProjectIndexFilePath($project->getRootFolder())
-        )->putContents($this->serialize($project));
+        return file_put_contents(
```

Comment:
It's because PHP main process will exit before the file stream flushed. In my case, the 27mb index file never got the chance to be written to disk under php 7.1 MacOS 10.12.2.